locale/de_DE/editor.xml

```diff
@@ -57,8 +57,8 @@
 	<message key="editor.monograph.legend.uploaded">Datei wurde hochgeladen durch die im Spaltentitel angegebene Rolle</message>
 
 	<!-- Editorial related -->
-	<message key="editor.submission.introduction">Im Stadium 'Einreichung' bestimmt der/die zuständige Herausgeber/in nach Prüfung der eingereichten Dateien das weitere Vorgehen, indem er/sie eine passende Aktion auswählt (der Autor/die Autorin wird darüber grundsätzlich informiert): Zur internen Begutachtung senden (anschließend müssen die für die interne Begutachtung vorgesehenen Dateien ausgewählt werden); Zur externen Begutachtung senden (anschließend müssen die für die externe Begutachtung vorgesehenen Dateien ausgewählt werden); Manuskript annnehmen (anschließend sind die Dateien für die redaktionelle Bearbeitung auszuwählen) oder Manuskript ablehnen (das Manuskript wird im Archiv abgelegt).</message>
-	<message key="editor.monograph.editorial.fairCopy">Reinschrift</message>
+	<message key="editor.submission.introduction">In der Einreichungsphase bestimmt der prüft der zuständige Herausgeber die eingereichten Manuskripte und prüft das weitere Vorgehen. Dazu wählt er/sie eine passende Aktion aus (über die der Autor/die Autorin grundsätzlich informiert wird): Zur internen Begutachtung senden (anschließend müssen die für die interne Begutachtung vorgesehenen Dateien ausgewählt werden); Zur externen Begutachtung senden (anschließend müssen die für die externe Begutachtung vorgesehenen Dateien ausgewählt werden); Manuskript annnehmen (anschließend sind die Dateien für die redaktionelle Bearbeitung auszuwählen) oder Manuskript ablehnen (das Manuskript wird im Archiv abgelegt).</message>
+	<message key="editor.monograph.editorial.fairCopy">Vom Autor genehmigte satzfertige Manuskriptfassung</message>
```

Comment:
This is something I think we've discussed before - it's just too long to identify a particular version. I am happy to discuss alternatives.